src/index.tsx

```diff
@@ -19,10 +19,11 @@ export type MutationNode<T extends OperationBase> = BaseMutationConfig<
   T
 >['mutation'];
 
-export type MutationConfig<T extends OperationBase> = WithOptionalFields<
-  Omit<BaseMutationConfig<T>, 'mutation'>,
-  'variables'
->;
+export type MutationConfig<T extends OperationBase> = Partial<
```

Comment:
i think `Partial` is essentially right here... empty objects are explicitly allowed